src/components/terminal/DockerTerminal.tsx

```diff
@@ -138,35 +130,119 @@ interface RouteParams {
   name: string;
 }
 
+enum FontSizeChangeType {
+  INCREASE = 'INCREASE',
+  DECREASE = 'DECREASE',
+  RESET = 'RESET',
+}
+
+const FONT_SIZE_DEFAULT = 12;
+const FONT_SIZE_MIN = 10;
+const FONT_SIZE_MAX = 20;
+
+const termOptions: ITerminalOptions = {
+  fontFamily: "source-code-pro, Menlo, Monaco, Consolas, 'Courier New', monospace",
+  fontSize: FONT_SIZE_DEFAULT,
+  lineHeight: 1.2,
+  cursorBlink: true,
+  cursorStyle: 'bar',
+  allowTransparency: true,
+  theme: nord,
+};
+
 const DockerTerminal: React.FC = () => {
   const { l } = usePrefixedTranslation('cmps.terminal.DockerTerminal');
   const { notify } = useStoreActions(s => s.app);
   const { type, name } = useParams<RouteParams>();
   const termEl = useRef<HTMLDivElement>(null);
+  const terminal = useRef<Terminal>();
+
+  const changeFontSize = (changeType: FontSizeChangeType) => {
+    if (
+      terminal.current &&
+      terminal.current.options &&
+      terminal.current.options.fontSize
+    ) {
+      const currentFontSize = terminal.current.options.fontSize;
+
+      switch (changeType) {
+        case FontSizeChangeType.INCREASE:
+          if (currentFontSize < FONT_SIZE_MAX) {
+            terminal.current.options.fontSize += 1;
+          }
+          break;
+        case FontSizeChangeType.DECREASE:
+          if (currentFontSize > FONT_SIZE_MIN) {
+            terminal.current.options.fontSize -= 1;
+          }
+          break;
+        case FontSizeChangeType.RESET:
+          terminal.current.options.fontSize = FONT_SIZE_DEFAULT;
+          break;
+        default:
+          break;
+      }
+    }
+  };
+
+  const contextMenuHandler = () => {
+    const menu = remote.Menu.buildFromTemplate([
+      { role: 'cut' },
+      { role: 'copy' },
+      { role: 'paste' },
+    ]);
+    menu.popup({ window: remote.getCurrentWindow() });
+  };
+
+  const keyupEventHandler = (e: KeyboardEvent) => {
+    if (e.ctrlKey) {
+      switch (e.key) {
+        case '+':
+          changeFontSize(FontSizeChangeType.INCREASE);
+          break;
+        case '-':
+          changeFontSize(FontSizeChangeType.DECREASE);
+          break;
+        case '0':
+          changeFontSize(FontSizeChangeType.RESET);
+          break;
+        case 'c':
+        case 'C':
+          if (e.shiftKey && terminal.current) {
+            clipboard.writeText((terminal.current.getSelection() || '').toString());
```

Comment:
The `.toString()` isn't necessary since you are defaulting to an empty string.

Reply:
Good point. Done.